### ISSUE TITLE
win_iis_webapplication: slashes in physical path are not converted to windows style

### ISSUE THREAD

**Issue by dagwieers** [ansible/ansible#25471]
_From @aensidhe on December 29, 2015 19:41_

##### ISSUE TYPE

Bug Report
##### COMPONENT NAME

win_iis_webapplication module
##### ANSIBLE VERSION

N/A
##### SUMMARY

win_iis_website converts them. Example:

vars somewhere:

```
path: c:/sites/https/develop
```

playbook:

```
- name: create website
  win_iis_website:
    name: "{{ name }}"
    state: started
    application_pool: "{{ appdomain }}"
    physical_path: "{{ path }}/web"

- name: create rest virtual folder
  win_iis_webapplication:
    name: rest
    state: present
    site: "{{ name }}"
    physical_path: "{{ path }}/api"
    application_pool: "{{ appdomain }}"
```

Website will work ok, webapplication will always fail with 404.

_Copied from original issue: ansible/ansible-modules-extras#1436_

**Comment by dagwieers** [comment]
_From @robynbergeron on January 11, 2016 17:37_

[module: windows/win_iis_webapplication.py]

**Comment by dagwieers** [comment]
_From @robynbergeron on January 11, 2016 18:50_

@aensidhe --Thanks for submitting this bug report.

@whenrik -- thanks in advance for taking a look at this bug report and for your ongoing work in maintaining this module.

**Comment by dagwieers** [comment]
_From @ansibot on August 4, 2016 20:50_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on September 13, 2016 3:37_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on October 6, 2016 18:14_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on October 22, 2016 10:20_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on November 6, 2016 10:55_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on November 22, 2016 11:3_

@whenrik, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on December 7, 2016 15:47_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
cc @jborean93 @jhawkesworth
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @nitzmahone
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @if-meaton
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @ShachafGoldstein
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jborean93** [comment]
Closing this issue due to it's age. If this is still a problem please post a new issue at https://github.com/ansible-collections/community.windows.
